src/lib/util/formatBytes/index.ts

```diff
@@ -0,0 +1,13 @@
+const k = 1024;
+const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
+
+export function formatBytes(bytes: number, decimals = 0): string {
```

Comment:
Based on this stack overflow answer:
https://stackoverflow.com/questions/15900485/correct-way-to-convert-size-in-bytes-to-kb-mb-gb-in-javascript

Which is based on this script here, but was updated to be ES6:
https://web.archive.org/web/20120507054320/http://codeaid.net/javascript/convert-size-in-bytes-to-human-readable-format-(javascript)